### ISSUE TITLE
BigDecimal with new Java does not work any more

### ISSUE THREAD

**Issue by MacX19** [odnoklassniki/one-nio#67]
Deserialization BigDecimal with Java 1.8.0_351 does not work any more:

```
java.io.IOException: readObject() is not fully supported. See implementation notes.
	at one.nio.serial.gen.NullObjectInputStream.unsupported(NullObjectInputStream.java:170)
	at one.nio.serial.gen.NullObjectInputStream.readFields(NullObjectInputStream.java:50)
	at java.math.BigDecimal.readObject(BigDecimal.java:3798)
	at sun.reflect.Delegate14_BigDecimal.read(Unknown Source)
	at one.nio.serial.GeneratedSerializer.read(GeneratedSerializer.java:127)
	at one.nio.serial.DeserializeStream.readObject(DeserializeStream.java:75)
...
```

**Comment by mikhailusachev** [comment]
Could you please provide a minimal testcase to reproduce?

**Comment by dormidon** [comment]
@MacX19 , thank you for the report and the example. 
We have tried to fix it in 1.6.1 release.
Could you please check if the issue is still actual or fixed?

**Comment by MacX19** [comment]
Looks like it's working properly now
